docs/guide_cog_creators.rst

```diff
@@ -0,0 +1,201 @@
+
+This guide serves to explain the Cog Creator Application process and lays out the requirements to be a Cog Creator.
+
+----------------------------------
+Creating a Cog Creator Application
+----------------------------------
+
+.. note::
+  You will need to have created and published your cogs before you create a Cog Creator Application!
+  See `guide_cog_creation` and `guide_publish_cogs` for more information.
+
+Cog Creator Applications are hosted on the `cogboard <https://cogboard.red/c/apps/12>`__.
+To create an application, start a new topic in the "Applications" category and fill out all of the required information.
+QA reviews Cog Creator Applications for security and functionality on a first come, first serve basis.
+Once your application is reviewed, you will have 14 days to make any requested changes, or to check in with the member of QA who is reviewing your application.
+
+-----------------------------
+Requirements for Cog Creators
+-----------------------------
+
+The following is a list of the requirements for approved Cog Creators.
+QA uses this list to request changes for Cog Creator Applications.
+Handling these requirements before submitting your application can streamline the review process.
+Any Cog Creator that does not follow these requirements will have their repo removed from approved listings any may have their Cog Creator status revoked.
+
+- Readme that contains
+
+  - Repository Name
+  - Installation Instructions
+  - Setup Instructions (if applicable)
+  - Credits (if applicable)
+
+- Repo-wide info.json with the keys
+
+  - ``author``
+  - ``name``
+  - ``short``
+  - ``description``
+
+- Cog info.jsons with the keys
+
+  - ``author``
+  - ``name``
+  - ``short``
+  - ``requirements`` (if applicable)
+  - ``description``
+
+  See `info-json-format` for more information on how to set up info.json files.
+
+- No cog contains malicious code.
+- No cog contains code that could impact the stability of the bot, such as blocking the event loop for an extended period of time, unreasonably high IO usage, etc.
+- No cog contains copied code that does not respect the license of the source.
+- Disclose in the install_msg info.json key of each cog that contains any of the following:
+
+  - Heavy memory or I/O usage.
+  - Any NSFW material.
+  - Bundled data.
+  - Stored data (outside of using Config).
+  - Interactions with outside services.
+  - Any extra setup instructions required.
+
+- No cog breaks the Discord TOS.
+- No cog conflicts with any core cogs (E.G., causing a core cog to fail to load) unless it is intended to replace that cog.
+- Your repo shows an understanding of a range of common cog practices. This requirement is to ensure QA can trust the safety and functionality of any future code you will create. This is handled on a case-by-case basis, however the following points should outline what we are looking to see:
+
+  - Cogs that are more than what is able to be run in an eval.
```

Comment:
Perhaps this was supposed to be "a simple eval" or something? I can do a lot of things in an eval...
```suggestion
  - Cogs that are more than what is able to be run in a simple eval.
```